src/axom/mint/CMakeLists.txt

```diff
@@ -118,7 +118,7 @@ set( mint_dependencies
      )
 
 blt_list_append( TO mint_dependencies
-                 ELEMENTS sidre conduit::conduit
+                 ELEMENTS sidre conduit::conduit hdf5
```

Comment:
(looking at sidre's cmake files) It looks like hdf5 is not a hard requirement of sidre.  Can you should probably guard it here as well.

sidres line:

```
blt_list_append(TO sidre_depends ELEMENTS hdf5 IF HDF5_FOUND)
```

Reply:
That's a good point -- I'll make that fix